README.md

```diff
@@ -10,21 +11,43 @@
 ember install ember-router-scroll
 ```
 
+## Basic use
```

Comment:
What do you think about moving the `Installation and Usage` section here?

Reply:
Sure no problem @snewcomer

Reply:
Sry.  I meant moving the entire section below (called `Installation and Usage`) and replace it with what you added (`Basic use`).  Lmk what you think.